### PR TITLE
Update scale script to

### PR DESCRIPTION
**Pull request by vishnoianil**
*.* Label containers with user id, so that multiple instances of the
script can be run and container belongs to each user can be identified
through label

*.* Add deployment flag to switch the URLs as per the deployment
environment, rather than changing it manually everytime.

*.* Fixing comments and usage text

### CODE REVIEW

File: hack/e2e-scripts/qa-container-scale.sh [hunk 1]
```diff
@@ -51,7 +51,7 @@ launch_containers() {
 
 # Function to print help message
 print_help() {
-  echo "Usage: ./script.sh -kc-password <password> -nexd-password <password> --nexd-count <count>"
+  echo "Usage: ./script.sh --kc-password <password> --nexd-password <password> --nexd-count <count>"
```
Comment (russellb) [inline hack/e2e-scripts/qa-container-scale.sh:54]: want to fix `script.sh` while you're at it? You could just replace it with `$0`

File: hack/e2e-scripts/qa-kind-container-scale.sh [hunk 2]
```diff
@@ -68,7 +68,7 @@ launch_containers() {
 
 # Function to print help message
 print_help() {
-  echo "Usage: ./script.sh -kc-password <password> -nexd-password <password> --nexd-count <count> [--custom-script]"
+  echo "Usage: ./script.sh --kc-password <password> --nexd-password <password> --nexd-count <count> [--custom-script]"
```
Comment (russellb) [inline hack/e2e-scripts/qa-kind-container-scale.sh:71]: here too

**Review by russellb — CHANGES_REQUESTED** [review]

**Review by nerdalert — APPROVED** [review]
Thanks @vishnoianil @russellb 

LGTM with RBs changes.

File: hack/e2e-scripts/qa-container-scale.sh (hunk 1 above)
Comment (vishnoianil) [reply hack/e2e-scripts/qa-container-scale.sh]: done

File: hack/e2e-scripts/qa-kind-container-scale.sh (hunk 2 above)
Comment (vishnoianil) [reply hack/e2e-scripts/qa-kind-container-scale.sh]: done

**Review by nerdalert — APPROVED** [review]

File: hack/e2e-scripts/qa-container-scale.sh [hunk 3]
```diff
@@ -51,13 +51,14 @@ launch_containers() {
...
 print_help() {
-  echo "Usage: ./script.sh -kc-password <password> -nexd-password <password> --nexd-count <count>"
+  echo "Usage: $0 --kc-password <password> --nexd-password <password> --nexd-count <count>"
   echo ""
   echo "Arguments:"
   echo "--kc-password <password>: The keycloak password for the kctool command."
   echo "--nexd-password <password>: The user password for the nexd command."
   echo "--nexd-count <count>: The number of nexd containers to launch and attach to the api-server."
-  echo "-help: Prints this help message."
+  echo "-d |--deployment <deployment>: The deployment to use. Valid values are prod,qa and playground. Default is qa."
```
Comment (russellb) [inline hack/e2e-scripts/qa-container-scale.sh:60]:
```suggestion
  echo "-d |--deployment <deployment>: The deployment to use. Valid values are prod, qa and playground. Default is qa."
```

File: hack/e2e-scripts/qa-container-scale.sh [hunk 4]
```diff
@@ -84,7 +85,22 @@ while (( "$#" )); do
...
+        "prod")
+           NEXODUS_API_SERVER="https://try.nexodus.io"
+           NEXODUS_AUTH_SERVER="https://auth.try.nexodus.io"
+           ;;
+        "playground")
+           NEXODUS_API_SERVER="https://playground.nexodus.io"
+           NEXODUS_AUTH_SERVER="https://auth.playground.nexodus.io"
+           ;;
+        *)
+           ;;
```
Comment (russellb) [inline hack/e2e-scripts/qa-container-scale.sh:99]: I'd prefer this checked for `qa`, and then under `*` it printed the help text and exited non-zero.

**Review by russellb — APPROVED** [review]
minor comments, but nothing worth blocking over

**Pull request merged**